.github/actions/generate-attestations/src/intoto.ts

```diff
@@ -0,0 +1,13 @@
+export const INTOTO_TYPE = "https://in-toto.io/Statement/v0.1";
+
+export type Subject = {
```

Comment:
Thanks for the PR @asraa. I will generate a json file containing an array of subjects of this type as the output of the `build` command, in the Docker builder.